sdk/communication/Azure.Communication.PhoneNumbers/src/PhoneNumbersClient.cs

```diff
@@ -0,0 +1,458 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Communication.PhoneNumbers.Models;
+using Azure.Communication.Pipeline;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Communication.PhoneNumbers
+{
+    /// <summary>
+    /// The Azure Communication Services phone numbers client.
+    /// </summary>
+    [CodeGenSuppress("GetByNumberAsync", typeof(string), typeof(CancellationToken))]
```

Comment:
Now that this is not generated, we probably do not need these `CodeGenSupress` series, right?

Reply:
The flag public-clients: true is still there to generate LROs.
We just suppress all not needed generated code and manually override the rest.

Reply:
Did we not agree to make this internal and write the public client manually?

Reply:
Made generated client internal and removed CodeGenSuppress